bitnami/keycloak/templates/statefulset.yaml

```
@@ -212,7 +214,7 @@ spec:
                   key: {{ include "common.secrets.key" (dict "existingSecret" .Values.auth.existingSecretPerPassword "key" "databasePassword") }}
                 {{- else }}
                   name: {{ include "keycloak.databaseSecretName" . }}
-                  key: "postgresql-password"
+                  key: "database-password"
```

Following my previous comment:
```suggestion
                  {{/*
                  We use a different key for PostgreSQL for backwards compatibility, see https://github.com/bitnami/charts/pull/8108
  TO BE DEPRECATED in 6.0.0
                  */}}
                  {{- if eq .Values.externalDatabase.vendor "postgres" }}
                  key: "postgresql-password"
                  {{- else }}
                  key: "database-password"
                  {{- end }}
```